doc/setup/enterprise.rst

```diff
@@ -0,0 +1,110 @@
+============================
+From Community to Enterprise
+============================
+
+Depending on your current installation, there are multiple ways to upgrade
+your community version.
+In any case the basic guidelines are:
+
+* Backup your community database
+
+  .. image:: enterprise/windows_db_manager.png
+    :class: img-responsive
+
+* Shutdown your server
+
+* Upgrade the web module
+
+* Restart your server
+
+* Enter your Odoo Enterprise Subscription code
+
+.. image:: enterprise/enterprise_code.png
+  :class: img-responsive
+
+On Linux, using an installer
+============================
+
+* Backup your community database
+
+* Stop the odoo service
+
+  .. code-block:: console
+
+    $ sudo service odoo stop
+
+* Install the enterprise .deb (it should install over the community package)
+
+  .. code-block:: console
+
+    $ sudo dpkg -i <path_to_enterprise_deb>, 
+  
+* Update your database to the enterprise packages using
+
+  .. code-block:: console
+
+    $ python /usr/bin/odoo.py -d <database_name> -u web --stop-after-init
+
+* You should be able to connect to your Odoo Enterprise instance using your usual mean of identification.
+  You can then link your database with your Odoo Enterprise Subscription by entering the code you received
+  by e-mail in the form input
+
+
+On Linux, using the source code
+===============================
+
+There are many ways to launch your server when using sources, and you probably
+have your own favourite. You may need to adapt sections to your usual workflow.
+
+* Shutdown your server
+* Backup your community database
+* Update the --addons-path parameter of your launch command (init.d, custom script,
+  configuration file, etc.). The Enterprise addons folder should be included ``before`` the community modules.
+  For example:
+
+  .. code-block:: console
+
+    # --addons-path=~/src/custom_modules,~/src/enterprise,~/src/odoo/addons
```

Comment:
replace the # by a $ otherwise some people may assume you need to do this as root